.github/workflows/blocked-prs.yml

```
@@ -203,6 +203,7 @@ jobs:
         run: |
           COMMENT_PATH="$(pwd)/temp_comment_file.txt"
           echo '<h3> PR Dependencies :pushpin:</h3>' > "$COMMENT_PATH"
```

```suggestion
          echo '<h3>PR Dependencies :pushpin:</h3>' > "$COMMENT_PATH"
```
This ever so slightly bugs me too